cmd/expired-authz-purger/main.go

```diff
@@ -41,19 +41,19 @@ type expiredAuthzPurger struct {
 	batchSize int64
 }
 
-func (p *expiredAuthzPurger) purgeAuthzs(purgeBefore time.Time, yes bool) (int64, error) {
+func (p *expiredAuthzPurger) purge(table string, yes bool, purgeBefore time.Time) error {
 	if !yes {
 		var count int
-		err := p.db.SelectOne(&count, `SELECT COUNT(1) FROM pendingAuthorizations AS pa WHERE expires <= ?`, purgeBefore)
+		err := p.db.SelectOne(&count, `SELECT COUNT(1) FROM  AS pa WHERE expires <= ?`, purgeBefore)
```

Comment:
It looks like the `table` is missing from this `SelectOne`. There's nothing after the `FROM` token.